### PR TITLE
feat: apisixcluster translation

### PR DESCRIPTION
**Pull request by tokers**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [ ] Bugfix
- [ ] New feature provided
- [ ] Improve performance
- [ ] Backport patches

- Related issues

___
### Bugfix
- Description

- How to fix?

___
### New feature or improvement
- Describe the details and related test reports.

___
### Backport patches
- Why need to backport?

- Source branch

- Related commits and pull requests

- Target branch

### CODE REVIEW

**Comment by tokers** [comment]
Ping @gxthrj @tao12345666333

**Review by tao12345666333 — APPROVED** [review]
LGTM

File: pkg/kube/translation/global_rule.go [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+}
+
+func (t *translator) TranslateClusterConfig(acc *configv2alpha1.ApisixClusterConfig) (*apisixv1.GlobalRule, error) {
+	globalRule := &apisixv1.GlobalRule{
+		ID:      id.GenID(acc.Name),
+		Plugins: make(apisixv1.Plugins),
+	}
+
+	if acc.Spec.Monitoring != nil {
+		if acc.Spec.Monitoring.Prometheus.Enable {
```
Comment (gxthrj) [inline pkg/kube/translation/global_rule.go:36]:
```suggestion
		if if acc.Spec.Monitoring.Prometheus != nil && acc.Spec.Monitoring.Prometheus.Enable {
```

Comment (tokers) [reply pkg/kube/translation/global_rule.go:36]: It's not a pointer, see https://github.com/apache/apisix-ingress-controller/blob/master/pkg/kube/apisix/apis/config/v2alpha1/types.go#L274.

**Review by gxthrj — APPROVED** [review]

**Pull request merged**